lib/modules/datasource/aws-machine-image/index.ts

```diff
@@ -11,8 +11,10 @@ import { Datasource } from '../datasource';
 import type { GetReleasesConfig, ReleaseResult } from '../types';
 import type { AwsClientConfig, ParsedConfig } from './types';
 
+const id = 'aws-machine-image';
```

Comment:
why this change? if it's required then please do in separate PR, otherwise this PR is hard to review because of big changes

Reply:
Stage 3 decorators seem to be instantiated before the class itself, so static constants are not available

Reply:
ok, please split that refactor to a separate PR, so we've less changes.